mitmproxy/proxy/mode_servers.py

```diff
@@ -122,6 +122,21 @@ async def stop(self) -> None:
     def listen_addrs(self) -> tuple[Address, ...]:
         pass
 
+    @property
+    @abstractmethod
+    def log_desc(self) -> str:
```

Comment:
Maybe we should use this opportunity and rename it to `description`, as in it's JSON. (And also because it's not just used in logging anymore.)

Reply:
👍